cuqi/distribution/_joint_distribution.py

```diff
@@ -175,6 +175,10 @@ def _parse_args_add_to_kwargs(self, *args, **kwargs):
                 kwargs[ordered_keys[index]] = arg
         return kwargs
 
+    def _sum_evaluated_densities(self):
+        """ Return the sum of the evaluated densities in the joint distribution """
+        return sum([density.logd() for density in self._evaluated_densities])
+
     def _reduce_to_single_density(self):
```

Comment:
Do we also need to update the case when we have `MultipleLikelihoodPosterior` to add constants?

Reply:
No because MultipleLikelihoodPosterior subclasses from JointDistribution. Therefore it knows and will potentially have EvaluatedDensities. Note the line ``return MultipleLikelihoodPosterior(*self._densities)`` all densities are returned. This is different than the single density cases, where we dont have a datastructure to store more than e.g. prior and likelihood.

Reply:
got it! thank you for the clarification